pallets/parachain-staking/src/weights.rs

```diff
@@ -133,50 +131,50 @@ impl<T: frame_system::Config> WeightInfo for SubstrateWeight<T> {
 	// Storage: ParachainStaking InflationConfig (r:1 w:1)
 	#[rustfmt::skip]
 	fn set_staking_expectations() -> Weight {
-		Weight::from_ref_time(27_136_000 as u64)
+		Weight::from_ref_time(48_225_000 as u64)
```

Comment:
This looks significant in change but could be a fluke. Did we change this code part?